zingolib/Cargo.toml

```diff
@@ -37,7 +37,7 @@ tower-http = { version = "0.2", features = ["add-extension"] }
 futures = "0.3.15"
 hex = "0.3"
 ring = "0.17.0"
-json = "0.12.4"
+json = { workspace = true }
```

Comment:
why add to the workspace if this is the only crate that is using the workspace dependency?

Reply:
so that zingo-status can use it